ts/components/messages/MessageDetailData.tsx

```diff
@@ -86,6 +86,10 @@ class MessageDetailData extends React.PureComponent<Props> {
   };
 
   public render() {
+    const serviceName = pot.getOrElse(this.props.serviceDetail, undefined)
+      ?.service_name;
+    const textToCopy = `${serviceName} - ${this.props.message.id}`;
```

Comment:
This may result in `undefined - 000001` which makes us look very sloppy.
I'd rather fall back to the _status quo_ that's it just the message ID.

```
const textToCopy: string = pot
      .toOption(this.props.serviceDetail)
      .map(({ service_name }) => `${service_name} - ${this.props.message.id}`)
      .getOrElse(this.props.message.id);
```

Reply:
Changes applied as suggested 👌🏻